CHANGELOG.md

```diff
@@ -1,5 +1,72 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.6.0
+
+<!-- generated comparing v5.5.3..master -->
+
+_Apr 5, 2022_
+
+A big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🧰 Update peer deps to support React 18 (#32063) @eps1lon
```

Comment:
```suggestion
- 🧰 Update peer dependencies to support React 18 (#32063) @eps1lon
```
It took me a few seconds to realize "deps" was short for dependencies 😅 Suggesting changes to avoid confusion!